### ISSUE TITLE
WCAG: Contrast on our focus states

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#232]
### Description of the bug

When using a keyboard to navigate a page it is important to have a good focus indicator so the user knows where the focus is at any point. We have got feedback that the exit button and our primary button should have better focus states. 

[Read more on WCAG 1.4.11 ](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

Primary:
![Skjermbilde 2022-06-17 kl  12 49 47](https://user-images.githubusercontent.com/33222679/174285759-5e425b9f-0dc9-4e83-8001-b7a7b49b3d21.png)

Exit button:
![Skjermbilde 2022-06-17 kl  12 49 20](https://user-images.githubusercontent.com/33222679/174285776-db86a6a4-e61e-4dfc-aa1c-92979f3e76fa.png)

## tasks
- [ ] Are there more focus states we should have look on while we are at it?
- [ ] Design new focus states
- [ ] implement

### Steps To Reproduce

Go to any app and navigate using a keyboard.

**Comment by bjosttveit** [comment]
Navigation buttons can easily be fixed by using design system buttons instead of the `AltinnButton` which is currently used. The design system buttons have much clearer focus states.

![Image](https://user-images.githubusercontent.com/47412359/214056733-a547f954-b251-4185-b139-34f0d1dc3957.png)

The exit button could maybe use the quiet variant of the design system button.

In that case, there would be no need to design new focus states.

Thoughts @Febakke ?

**Comment by bjosttveit** [comment]
I can do the low hanging fruit first, like using the design system buttons where applicable (navigation, submit, possibly exit-button?)

Could you make a clear list of which components are problematic @Febakke ?
